metabase/metabase
# Upgrade `documentation` lib

### Status
PENDING CI && PENDING REVIEW

### What does this PR accomplish?
Upgrades badly outdated `documentation` library to the latest version, which seems to solve [this error](https://github.com/metabase/metabase/pull/17137#issuecomment-884163661) in the `upgrade-babel` attempt.

### How to verify it works?
- run `yarn docs`
- open `localhost://4001` and make sure everything is in order

alternatively
- run `yarn dev` or `yarn dev-ee`
- make sure documentation is served on `localhost://4001` and make sure the above mentioned error is not present anymore
